install_config/aggregate_logging.adoc

```diff
@@ -65,9 +63,7 @@ $ oadm new-project logging --node-selector label=value
 +
 [NOTE]
 ====
-The selector in the example is optional. It can be used to specify a
-region or other selector you would like to use to restrict where the
-logging infrastructure deploys.
+Using node selectors is optional. These can be used to specify a region or other selector you would like to use to restrict the nodes onto which the logging infrastructure deploys.
```

Comment:
This second sentence is way confusing to me. I'd suggest a rewording, perhaps:

"They can specify a region (or other selector) by which to restrict the nodes where the the logging infrastructure deploys."